barbeque/static.py

```diff
@@ -16,17 +28,58 @@ def __call__(self, request):
         response = self.get_response(request)
         return self.process_response(request, response)
 
+    def serve_response(self, request, file_path):
+        return serve(request, file_path, document_root=settings.STATIC_ROOT)
+
+    def find_requested_file_hashed_name(self, requested_path):
+        # Load static files mapping manifest.
+        # It maps original names with hashed ones.
+        storage = ManifestStaticFilesStorage()
```

Comment:
I'd fetch the storage from settings in case a custom one is used that handles the manifest file in another way.